### PR TITLE
Ob2turns1 discount shopping post

### PR DESCRIPTION
**Pull request by jenncloud**

### CODE REVIEW

File: _posts/blog/2018-11-20/2018-11-20-big-discounts-when-You-shop-openbazaar-this-week.markdown [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+#### Here are some great stores and listings for you to shop this week:
+<br>
+#### DrApis
+_75% off_
+
+![DrApis on OpenBazaar](DrApis_on_OpenBazaar.png "DrApis on OpenBazaar")[DrApis on OpenBazaar](https://openbazaar.com/store/QmYTXDyMNjdUSvqNc88T2VeVF3KdG7PMefnGQKrp9NZ5Tp)
+<br>  
+<br>
+DrApis produces and sells premium raw apiculture products from Portugal. Their artisanal raw honey is produced and extracted using exclusively traditional methods, preserving all the natural vitamins, enzymes, phytonutrients and other nutritional elements produced by bees. Now you can enjoy their high-quality products at their lowest prices _ever._ Don't miss this!
```
Comment (jjeffryes) [inline _posts/blog/2018-11-20/2018-11-20-big-discounts-when-You-shop-openbazaar-this-week.markdown]: Is is apiculture or apicultural?

File: _posts/blog/2018-11-20/2018-11-20-big-discounts-when-You-shop-openbazaar-this-week.markdown [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+<br>  
+<br>
+#### Matthew Zipkin
+_75% off_
+
+![Matthew Zipkin Bitcoin Clocks on OpenBazaar](Matthew_Zipkin_on_OpenBazaar.png "Matthew Zipkin Bitcoin Clocks on OpenBazaar")[Matthew Zipkin on OpenBazaar](https://openbazaar.com/store/QmeSyTRaNZMD8ajcfbhC8eYibWgnSZtSGUp3Vn59bCnPWC)
+<br>  
+<br>  
+Matthew Zipkin is offering this steep discount on his incredibly neat Bitcoin clocks! has some neat, practical art pieces available in his store.
```
Comment (jjeffryes) [inline _posts/blog/2018-11-20/2018-11-20-big-discounts-when-You-shop-openbazaar-this-week.markdown]: > clocks! has some

Is there a missing "He" here?

File: _posts/blog/2018-11-20/2018-11-20-big-discounts-when-You-shop-openbazaar-this-week.markdown [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+Another fun fact, Matthew is also an [OB1 Verified Moderator](https://ob1.io/verified-moderators.html) who is available for you to add to your transactions and provide assistance in case of a dispute. Send him a message and say hello!
+<br>  
+<br>  
+#### CryptoCollectibles
+_75% off_
+
+![CryptoCollectibles on OpenBazaar](CryptoCollectibles_on_OpenBazaar.png "CryptoCollectibles on OpenBazaar")[CryptoCollectibles on OpenBazaar](https://openbazaar.com/store/QmakxNv9Y5YFuAuvjvx4mEda3TmFBaHDtacd1KtEL56mP4/)
+<br>  
+<br> 
+CryptoCollectibles has curated several items to make available at this crazy discount. In honor of his recent passing they are all vintage Stan Lee comics. Some have art by Jack "The King" Kirby, Gene Colan, Gil Kane, and other forefathers of the comic book game. Check out the full list:
```
Comment (jjeffryes) [inline _posts/blog/2018-11-20/2018-11-20-big-discounts-when-You-shop-openbazaar-this-week.markdown]: > to make available

That's future tense, presumably they are available now.

I suggest "has curated several items that are available at this"

File: _posts/blog/2018-11-20/2018-11-20-big-discounts-when-You-shop-openbazaar-this-week.markdown [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+<br>  
+Discounts are just starting at 25% with Stock Media Selling. Follow the hashtag [#OB2Turns1](https://twitter.com/hashtag/OB2Turns1) on Twitter for their list.
+<br>  
+<br>  
+#### Pascal Boyart 
+_10% off_
+![Pascal Boyart on OpenBazaar](Pascal_Boyart_on_OpenBazaar.png "Pascal Boyart on OpenBazaar")[Pascal Boyart on OpenBazaar](https://openbazaar.com/store/QmYTXDyMNjdUSvqNc88T2VeVF3KdG7PMefnGQKrp9NZ5Tp)
+<br>  
+<br>  
+Pascal Boyart is an accomplished fine artist based in Paris. He began as a graffiti artist but now produces a wide range of art styles. Many of the pieces he has featured on OpenBazaar focus on cryptocurrency and crypto-culture. Read more of his amazing story on his [About page](https://openbazaar.com/store/home/QmYTXDyMNjdUSvqNc88T2VeVF3KdG7PMefnGQKrp9NZ5Tp).
```
Comment (jjeffryes) [inline _posts/blog/2018-11-20/2018-11-20-big-discounts-when-You-shop-openbazaar-this-week.markdown]: Add a period after About page.

File: _posts/blog/2018-11-20/2018-11-20-big-discounts-when-You-shop-openbazaar-this-week.markdown (hunk 1 above)
Comment (jenncloud) [reply _posts/blog/2018-11-20/2018-11-20-big-discounts-when-You-shop-openbazaar-this-week.markdown]: They say apiculture in their description!

**Pull request merged**